app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user, class_name: 'User', foreign_key: 'users_id'
```

Comment:
A post should belong to an `author` instead of a user.

Reply:
Hi @TheLuisAbarca , thanks for reviewing my job and for your kind feedback. The changes are implemented in commit 90ae214.